pkg/ebpftracer/tracer.go

```diff
@@ -253,8 +254,8 @@ func (t *Tracer) runPerfBufReaderLoop(ctx context.Context, target *ebpf.Map) err
 		if err := t.decodeAndExportEvent(ctx, ebpfMsgDecoder); err != nil {
 			if errors.Is(err, decoder.ErrTooManyArguments) {
 				data := ebpfMsgDecoder.Buffer()
-				t.log.Errorf("decoding event: too many arguments for event. payload: %s",
-					base64.StdEncoding.EncodeToString(data))
+				t.log.Errorf("decoding event: too many arguments for event. payload=%s, err=%v",
```

Comment:
Why do you want to log the err here as well? It will always be `ErrToManyArguments`.

Reply:
Ah ok, you wrapped it, so it will contain the `EventID`, nice!